src/styles/blocks/stopwatch.scss

```
@@ -0,0 +1,68 @@
+.stopwatch {
+  position: relative;
+  width: $width;
+  height: $width;
+  border: $border-width dotted $border-color;
+  border-radius: 50%;
+
+  &__minutes-hand {
+    position: absolute;
+    height: $minutes-hand-height;
+    width: $hand-width-minutes;
+    background-color: $minutes-hand-color;
+    bottom: 50%;
+    left: calc(50% - (#{$hand-width-minutes} / 2));
+    transform-origin: bottom center;
+    animation: spin-minute 3600s steps(60) infinite;
+  }
+
+  &__seconds-hand {
+    position: absolute;
+    height: $seconds-hand-height;
+    width: $hand-width-seconds;
+    background-color: $seconds-hand-color;
+    bottom: 50%;
+    left: calc(50% - (#{$hand-width-seconds} / 2));
+    transform-origin: bottom center;
+    animation: spin-second 60s linear infinite;
+  }
+
+  &__center {
+    position: absolute;
+    width: $center-size;
+    height: $center-size;
+    border-radius: 50%;
+    background-color: $center-color;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+  }
+
+  &--speed-up {
+    .stopwatch__seconds-hand {
+      animation-duration: 10s;
+    }
+
+    .stopwatch__minutes-hand {
+      animation-duration: 600s;
+    }
+  }
+}
+
+@keyframes spin-minute {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+@keyframes spin-second {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
```

The scenario are same, so you need to use only one keyframes
```suggestion
@keyframes rotate {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}
```